v2/Cargo.toml

```diff
@@ -27,7 +27,10 @@ chrono = { version = "0.4.31", features = ["serde"] }
 env_logger = { workspace = true }
 futures = "0.3.28"
 git-version = { workspace = true }
-influxdb2 = { version = "0.4.5", features = ["rustls"], default-features = false }
+humantime = "2.1.0"
```

Comment:
Chrono is already a dependency and can do RFC3339 formatting with nanoseconds precision.
Better to use it than to import an additional dependency for this.

Reply:
Added the changes.